auth-backends/README.adoc

```diff
@@ -543,7 +586,7 @@ In the log, you see:
 It means that the value used for `rudder.auth.oauth2.provider.${registrationKey}.userNameAttributeName` was correctly returned in the profile list for the authenticated user, but that value was not found in Rudder user configuration files `/opt/rudder/etc/rudder-users.xml`.
 Check that one of the entries in that file has the corresponding value for its `name` attribute.
 
-=== Radius backend
+=== Radius backend (deprecated)
```

Comment:
"deprecated" is already in a warn info notice just below, but some people were still missing it - nothing to do with the main pr, it's just cleaning things up like other typos.